docs/ESLINT_PLUGIN_TSLINT.md

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
why not use the package's readme?

Reply:
generally this is just placeholdder, we should explain a little more how to use this package, and reusing readme is not great idea